### PR TITLE
Adding Krystan HuffMenne to the Ember Data core team

### PR DESCRIPTION
**Pull request by kiwiupover**
This PR adds Krystan HuffMenne to the Ember Data core team
and moves Scott Newcomber to alumni

### CODE REVIEW

**Review by wagenet — APPROVED** [review]

**Review by runspired — APPROVED** [review]

**Review by IgnaceMaes — APPROVED** [review]
Thanks Scott for your contributions 🙌 

And congrats Krystan 👏

**Review by IgnaceMaes — CHANGES_REQUESTED** [review]
Looks like two tests are failing. If you could make a quick adjustment to the assertions that would be great. Otherwise I can get back to it when I'm on a computer.

File: tests/unit/controllers/teams-test.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ module('Unit | Controller | teams', function (hooks) {
     // Create an intermediate data structure for assertion
     const output = dataTeamMembers.map((teamMember) => teamMember.id);
 
-    assert.deepEqual(output, ['chris-thoburn', 'scott-newcomer']);
+    assert.deepEqual(output, ['chris-thoburn', 'krystan-huffMenne']);
```
Comment (gitKrystan) [inline tests/unit/controllers/teams-test.js:68]:
```suggestion
    assert.deepEqual(output, ['chris-thoburn', 'krystan-huffmenne']);
```

**Review by IgnaceMaes — APPROVED** [review]

**Comment by IgnaceMaes** [comment]
Thanks!

**Pull request merged**